api/base/utils.py

```diff
@@ -17,6 +17,15 @@
 TRUTHY = set(('t', 'T', 'true', 'True', 'TRUE', '1', 1, True))
 FALSY = set(('f', 'F', 'false', 'False', 'FALSE', '0', 0, 0.0, False))
 
+UPDATE_METHODS = ['PUT', 'PATCH']
+
+def is_bulk_request(request):
+    """
+    Returns True if bulk request.  Can be called as early as the parser.
+    """
+    content_type = request.content_type
+    return 'ext=bulk' in content_type
```

Comment:
This could break if other extensions are used, e.g. `ext=profile,bulk`, but I think we can keep this simple for now. Being able to gracefully change this implementation is the reason we created this function in the first place.